human.descriptions.py

```diff
@@ -3474,11 +3761,28 @@ def main():
             'bn': { 'male': '~ মডেল', 'female': '~ মডেল' },
             'en': { 'male': '~ model', 'female': '~ model' },
             'es': { 'male': 'modelo ~', 'female': 'modelo ~' },
+            'et': { 'male': '~ modell', 'female': '~ modell' },
             'fr': { 'male': 'mannequin ~', 'female': 'mannequin ~' },
             'he': { 'male': 'דוגמן ~', 'female': 'דוגמנית ~' },
-            'es': { 'male': 'model ~', 'female': 'model ~' },
+            'ro': { 'male': 'model ~', 'female': 'model ~' },
```

Comment:
I'm fairly sure this second "es" was supposed to be "ro" - I made sure that Romanian uses "model" before making this change